.github/workflows/ci.yml

```diff
@@ -700,10 +700,10 @@ jobs:
             values: 16-validators
             runner_type: 16core-tester-x86
             timeout: 60
-          # TODO(#9736)
-          # - test: transfer.test.ts
-          #   values: 48-validators
-          #   runner_type: 32core-tester-x86
+          # TODO(#9736) this fails
+          - test: transfer.test.ts
+            values: 48-validators
+            runner_type: 32core-tester-x86
```

Comment:
Is the "this fails" comment still relevant?

Reply:
I don't know, I didn't actually uncomment this